ui/helpers/utils/util.js

```diff
@@ -631,3 +635,58 @@ export const getNetworkNameFromProviderType = (providerName) => {
 export const isAbleToExportAccount = (keyringType = '') => {
   return !keyringType.includes('Hardware') && !keyringType.includes('Snap');
 };
+
+/**
+ * Converts hex string to decimal value.
+ *
+ * @param {*} hex - string hex to convert to decimal
+ * @returns string decimal
+ */
+
+export const hexToDecimal = (hex) => parseInt(hex, 16);
+
+/**
+ * Converts decimal string to hex string
+ *
+ * @param {*} dec - string to convert to hex
+ * @returns string hex
+ */
+export const decimalToHex = (dec) => {
+  const decimalNumber = parseInt(dec, 10); // 10 is the base for decimal
+  return `0x${decimalNumber.toString(16)}`;
+};
+
+/**
+ * Checks if a tokenId in Hex or decimal format already exists in an object.
+ *
+ * @param {*} address - collection address.
+ * @param {*} tokenId - tokenId to search for
```

Comment:
Although this file is not typescript, more specific types could be used in the docs:
```suggestion
 * @param {string} address - collection address.
 * @param {string} tokenId - tokenId to search for
```